xbmc/interfaces/legacy/ListItem.cpp

```diff
@@ -588,6 +588,30 @@ namespace XBMCAddon
       }
     } // end ListItem::setInfo
 
+    void ListItem::setCast(const std::vector<Properties>& actors)
+    {
+      LOCKGUI;
+      item->GetVideoInfoTag()->m_cast.clear();
+      for (const auto& dictionary: actors)
+      {
+        SActorInfo info;
+        for (auto it = dictionary.begin(); it != dictionary.end(); ++it)
+        {
+          const String& key = it->first;
+          const String& value = it->second.c_str();
```

Comment:
that´s how i did it in the beginning, but changed it based on @tamland comment (see above)
So should I change it back?